fs/info.py

```diff
@@ -71,6 +69,15 @@ def get(self, namespace, key, default=None):
         except KeyError:
             return default
 
+    def _require_namespace(self, namespace):
```

Comment:
This looks like a good idea to me. Indeed the `isdir` and `isfile` worked because the `is_dir` attribute was in the *basic* namespace, but since the `is_link` cannot be forced into it, that's the only good solution for me. 

Although I'm not sure to understand: why 3 return values ?